pulsar-io/nifi/src/main/java/org/apache/pulsar/io/nifi/README.md

```diff
@@ -0,0 +1,58 @@
+<!--
```

Comment:
I would suggest

1) moving the README.md to `pulsar-io/nifi`
2) adding the same documentation to website as well.

Reply:
> I would suggest
> 
> 1. moving the README.md to `pulsar-io/nifi`
> 2. adding the same documentation to website as well.

ok